src/tests/ftest/server/multiengine_persocket.py

```diff
@@ -201,6 +199,8 @@ def test_multiengines_per_socket(self):
         self.daos_server_scm_reset(step)
         engines_per_socket = self.params.get(
             "engines_per_socket", "/run/server_config/*", default=1)
+        reboot_after_format = self.params.get(
+            "reboot_after_format", "/run/server_config/*", default=False)
         self.daos_server_scm_prepare_ns(step, engines_per_socket)
```

Comment:
@dinghwah let's try this just to see if we can get it behaving consistently, then we can tidy up after. The multiple reboots will mean we clear and recreate the PMem regions as well as namespaces, hopefully this will ensure the naming of the namespaces ndctl create are consistent each time.
```suggestion
        self.daos_server_scm_reset(step)
        time.sleep(15)
        self.host_reboot(self.hostlist_servers)
        engines_per_socket = self.params.get(
            "engines_per_socket", "/run/server_config/*", default=1)
        self.daos_server_scm_prepare_ns(step, engines_per_socket)
        time.sleep(15)
        self.host_reboot(self.hostlist_servers)
        self.daos_server_scm_prepare_ns(step, engines_per_socket)
        time.sleep(15)
        reboot_after_format = self.params.get(
            "reboot_after_format", "/run/server_config/*", default=False)
```

Reply:
If it is needed 2 "daos_server  scm prepare --force --scm-ns-per-socket=2" ?

Reply:
I updated the script with following steps: (2  self.daos_server_scm_prepare_ns) the pmem0.1 and pmem1.1 came up consistently.
step (1.A): daos_server scm reset --force
                 server reboot  <-----  This reboot can be removed.
step (1.1B): daos_server scm prepare –force --scm-ns-per-socket 2
                 server reboot
step (1.2B): daos_server scm prepare –force --scm-ns-per-socket 2

Reply:
Please see the newly updated script. 
Thanks.